### DISCUSSION TITLE
SFCC dynamic sandbox

### DISCUSSION THREAD

**Discussion by sompra** [SalesforceCommerceCloud/sfcc-ci#548] [Team Posts]
Hi team,

Do we know if the functionality of SFCC dynamic sandbox is released or not ? 
If yes, how can we get access to it ?

Thanks in advance.

**Comment by w6bedi** [comment]
Hi,
On demand sandboxes are in beta right now and will go GA in August this year. 

If you are interested to still join beta, let me know. 

Regards,
Dirk

**Comment by alawrenceSF** [comment]
Salesforce B2C On-Demand Developer Sandboxes are scheduled for GA in August. The beta program is currently open for new participants to sign on through the end of May and the program will end on July 31.

**Comment by sompra** [comment]
Hi, @w6bedi yes I’m interested to join the beta, how can I be part of it ?
Thanks

**Comment by fishme** [comment]
Yeah me too where I can register?

**Comment by raokotagiri** [comment]
Me too, Is there a place that i can register

Thanks,

Partha Kotagiri

**Comment by divgo** [comment]
If these are the Container Instance Sandboxes that you are referring to, then you need to reach out to your account manager and request to be included in the Preview. It took us a few months, but we recently got invited to this preview. They sent us a contract addendum that we had to sign and return first.
